src/ceph-volume/ceph_volume/tests/util/test_util.py

```diff
@@ -75,22 +75,22 @@ class TestPromptBool(object):
     def test_trueish(self, response):
         fake_input = lambda x: response
         qx = 'what the what?'
-        assert util.prompt_bool(qx, _raw_input=fake_input) is True
+        assert util.prompt_bool(qx, _input=fake_input) is True
```

Comment:
Doesn't this file also need the `from six.moves import input` import to function correctly on python 2?